validator/capability.go

```diff
@@ -1,8 +1,12 @@
 package validator
```

Comment:
Reading this in comparison to ucanto, as I understand it a lot of logic moved to the policy  module in core, which I understand to be ucan 1.0 compatible. Is this validator also js-ucanto compatible? IOW has the DSL changed significantly?

Reply:
also, should we go ahead and port the delegation logic to js ucanto?

Reply:
Yep - there's a PR open for it that did not get finished yet! https://github.com/storacha-network/ucanto/pull/344